frontend/components/Domain/Recipe/RecipePage/RecipePageParts/RecipePageInstructions.vue

```diff
@@ -123,7 +147,7 @@
               <v-card-title :class="{ 'pb-0': !isChecked(index) }">
                 <span :class="isEditForm ? 'handle' : ''">
                   <v-icon v-if="isEditForm" size="26" class="pb-1">{{ $globals.icons.arrowUpDown }}</v-icon>
-                  {{ $t("recipe.step-index", { step: index + 1 }) }}
+                  {{ step.summary ? step.summary : $t("recipe.step-index", { step: index + 1 }) }}
                 </span>
                 <template v-if="isEditForm">
```

Comment:
This should allow the user to directly edit the step summary. 

```suggestion
                <v-text-field
                  v-if="isEditForm"
                  v-model="step.summary"
                  class="headline handle"
                  hide-details
                  dense
                  solo
                  flat
                  :placeholder="$t('recipe.step-index', { step: index + 1 })"
                >
                <template #prepend>
                  <v-icon size="26">{{ $globals.icons.arrowUpDown }}</v-icon>
                </template>
                </v-text-field>
                <span v-else>
                  {{ step.summary ? step.summary : $t("recipe.step-index", { step: index + 1 }) }}
                </span>
```